.gitignore

```diff
@@ -27,3 +27,5 @@ dist/
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules/
 bower_components/
+
+**/*.DS_Store
```

Comment:
Definitely a weird, automatically-generated OSX thing.  I looked into it and I think we should actually keep it in the gitignore.  Check out the wikipedia wisdom:
".DS_Store files impose additional burden on revision control process: They are prone to changes and therefore appear in commits unless specifically excluded."
What you think?